packages/lib/time.ts

```diff
@@ -89,8 +89,8 @@ class Time {
 		);
 	}
 
-	public unixMsToLocalDateTime(ms: number): string {
-		return moment.unix(ms / 1000).format('DD/MM/YYYY HH:mm');
+	public unixMsToLocalDateTime(ms: number, format?: string): string {
```

Comment:
So as you can see in this file, there's already a `formatMsToLocal()` method that does the same thing and that is already setup to use the preferred date/time format. Can this be used instead?

Reply:
Ah, that was an oversight on my part. I basically modified `unixMsToLocalDateTime` to do what `formatMsToLocal` already does. I have made the changes to use `formatMsToLocal` instead.